packages/sources/cfbenchmarks/src/endpoint/birc.ts

```diff
@@ -0,0 +1,44 @@
+import { InputParameters, Requester, Validator } from '@chainlink/ea-bootstrap'
+import { ExecuteWithConfig } from '@chainlink/ea-bootstrap'
+import { Config } from '../config'
+
+export const supportedEndpoints = ['birc']
+
+export type TInputParameters = Record<string, never>
+export const inputParameters: InputParameters<TInputParameters> = {}
```

Comment:
This endpoint will need to be added to [cfbenchmarks-test](https://github.com/smartcontractkit/external-adapters-js/tree/develop/packages/sources/cfbenchmarks-test) as well before the EAv3 version can go live.

Reply:
Working on that now 👍